### PR TITLE
initial

### PR DESCRIPTION
**Pull request by jquense**
there was so much git weirdness

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  'last 2 Safari versions',
+];
+
+function preset(_, options = {}) {
+  const env = process.env.NODE_ENV || 'production'; // default to prod
+  const opts = Object.assign({}, defaultOptions, options);
+  const target = opts.target;
+
+  const nodeTarget = {
+    node: env === 'production' ? '8.3' : 'current',
```
Comment (jquense) [inline index.js]: 8.3 adds object rest/spread

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (taion) [inline .gitignore]: That's all you need now? Maybe better to use https://github.com/github/gitignore/blob/master/Node.gitignore anyway?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      require.resolve('@babel/plugin-proposal-class-properties'),
+      // -----
+
+      // - stage 1 --
+      require.resolve('@babel/plugin-proposal-export-default-from'),
+      require.resolve('@babel/plugin-proposal-export-namespace-from'),
+      // -----
+
+      // - convenience plugins --
+      require.resolve('babel-plugin-dev-expression'),
```
Comment (taion) [inline index.js]: This didn't need an update to Babel 7?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  "description": "4C babel preset",
+  "main": "index.js",
+  "repository": "https://github.com/4Catalyzer/babel-preset-4catalyzer.git",
+  "author": "Jason Quense <monastic.panic@gmail.com>",
+  "license": "MIT",
+  "engines": {
+    "node": ">= 8.3.0"
+  },
+  "scripts": {
+    "precommit": "lint-staged"
```
Comment (taion) [inline package.json]: add a lint script?

File: index.js (hunk 3 above)
Comment (jquense) [reply index.js]: i'll double check but i don't think so.

Comment (jquense) [reply index.js]: ya works fine

**Pull request merged**